product/outline.md

```diff
@@ -0,0 +1,79 @@
+*  understand additional negotiation possibilities that InnerSource brings - keeping the balance of Give and Take with InnerSource
+
+# Outline
+
+InnerSource and Agile
+
+You want to improve your product and deliver those improvements faster to customers. You want to make stakeholders happy. InnerSource helps your team deliver value and maintain autonomy especially in a highly interconnected world.
+
+- Issues InnerSource solves
+   - avoid lengthy roadmap syncronization discussions: "help with solution" instead of "wait it out" or "build workarounds"
+   - resolve cross dependencies without compromising autonomy, code ownership and quality
+   - better support for remote first by higher focus on written communication
+
+
+- How InnerSource and Agile complement each other
+   - Well factored and well tested code leads to faster on-ramp for agile teams, but also helps InnerSource allowing for faster on-ramp for contributors
+   - A focus on working together and not assigning individual tasks allows for easier on-ramp of contributors.
+   - Existing local pairing can make cross team pairing and sharing of work in progress easier.
+   - Existing test driven development means that confidence in contributions is higher.
+   - XP teams have a "leave the code in better shape than you found it" attitude - mitigates the risk of decline in quality or co-hesion of code when faced with multiple contributions from different sources.
+
+
+Challenges with InnerSource and agile (these challenges form the basis for the remaining segments in this section).
+
+- Common mis-understandings
+   - Common fears (like losing control) and counter measures
+   - Same phrase - different meaning: How issue trackers and code reviews are used differently in InnerSource projects than is common in the industry.
+   - The impact of transparency: How InnerSource depends on trust, what to avoid to keep trust alive.
+   
+
+InnerSource and capacity planning
+
+- Small vs. large changes
+- Planning for mentoring
+- Contributions as long term investments in your upstream InnerSource project
+Representing in roadmap.
+
+The need for negotiation skills in InnerSource projects
+
+- Small vs. large changes - why smaller changes are easier to integrate ad-hoc than large refactorings
+- Best practices for integrating large changesets
+   - Coordinate a common understanding of the InnerSource project roadmap
+   - Up-front product and engineering design review
+   - Coordinate the timing of larger contributions
+- Unconventional solutions to priority conflicts
+- Communicate the need to think in contributions instead of building workarounds.
+
+
+Options for shared ownership
+
+- Introduce InnerSource governance levels 
+   - Governance levels pattern
+   - Implications of each level on product management
+- Common hurdles (line management, performance review setups)
+- Advantages
+- Commons issues wrt. operations and solutions
+
+
+Relation to open source
```

Comment:
My thinking here was that certain goals can be achieved with InnerSource, but as soon as people start thinking about collaboration the step towards open source is fairly small. The idea is to highlight which aspects would be an argument to aim for open instead of InnerSource.

What I have in mind for that is a link to https://blog.mozilla.org/wp-content/uploads/2018/05/MZOTS_OS_Archetypes_report_ext_scr.pdf - page 28